core/src/main/java/org/keycloak/representations/oidc/OIDCClientRepresentation.java

```diff
@@ -122,12 +122,14 @@
     private String backchannel_logout_uri;
 
     private Boolean backchannel_logout_session_required;
-    
+
     private Boolean backchannel_logout_revoke_offline_tokens;
 
     // OIDC CIBA
```

Comment:
AFAIK, Client related entities are as follows.

- Representation
 It stands for the format in representating Client.
 This format is used when communicating between keycloak and other entities.
   - [1] ClientRepresentation
 It is Client's Representation.
 When we access Client by Admin REST API(namely, on Admin Console), this ClientRepresentation is used.

- Model
 It stands for the format used when keycloak's internal program treats Client.
  - [2]  ClientModel
 It is Client's Model

- Store
 It stands for the format used when keycloak's internal program stores Client.
   - Infinispan Cache
  Used when storing Client in memory
      - [3] ClientAdapter(infinispan)
   - JPA:
  Used when storing Client in persistent storage
      - [4] ClientAdapter(jpa)
      - [5] ClientEntity(jpa)

- OIDC Extension
 It extends Client so that it can be treated as OIDC conformant Client
   - Representation
     - [6] OIDCClientRepresentation
   It stands for the format in representating OIDC conformant Client.
   This format is used when communicating between keycloak and other entities in OIDC manner
   eg.
   When we access Client by OIDC Dynamic Client Registration, this OIDCClientRepresentation is used.

   - Representation/Model
      - [7] OIDCAdvancedConfigWrapper
    It works as the wrapper between ClientRepresentation and OIDCClientRepresentation
      - [8] DescriptionConverter
    It converts the Client representation between ClientRepresentation and OIDCClientRepresentation.
      - [9] OIDCConfigAttributes
     It define keys used to store/access OIDC related parameter to ClientRepresentation's map object called Attribute and ClientModel's map object also callled Attribute.

- UI
   - [10] client.js
   - [11] client-detail.html
   - [12] admin-messages_en.properties 

In this PR, we need not to modify [2][3][4][5]. We need to modify [1][6][7][8][9][10][11][12].
You've already modified [1][6][7][9][10][11][12]. However, it seems that you've misssed [8].

[8] is needed when we modify `backchannel_user_code_parameter` by Admin REST API or Dynamic Client Registration.
When we test by Arquillian Integration Test, we need to modify `backchannel_user_code_parameter` being `true` to test this User Code support.

It might be helpful if you refer to codes about `token_delivery_mode` in DescriptionConverter.

Reply:
Changed this, thank you.

Reply:
I've confirmed. Thank you.